### PR TITLE
fix(plasma-ui): fix Pickers for React 18

### PR DESCRIPTION
**Pull request by akhdrv**
:baby_chick: Download canary assets:

[colors--canary.207.3384012578.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.207.3384012578.xml)  
[color_sbermarket_ios-swift--canary.207.3384012578.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.207.3384012578.swift)  
[color_sbermarket_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.207.3384012578.kt)  
[color_sbermarket_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.207.3384012578.ts)  
[color_sberprime_ios-swift--canary.207.3384012578.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.207.3384012578.swift)  
[color_sberprime_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.207.3384012578.kt)  
[color_sberprime_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.207.3384012578.ts)  
[PlasmaTokensColor--canary.207.3384012578.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.207.3384012578.swift)  
[typo_mage_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.207.3384012578.kt)  
[typo_mage_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.207.3384012578.ts)  
[typo_plasma_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.207.3384012578.kt)  
[typo_plasma_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.207.3384012578.ts)  
[typo_ruler_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.207.3384012578.kt)  
[typo_ruler_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.207.3384012578.ts)  
[typo_sage_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.207.3384012578.kt)  
[typo_sage_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.207.3384012578.ts)  
[typo_sbermarket_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.207.3384012578.kt)  
[typo_sbermarket_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.207.3384012578.ts)  
[typo_soulmate_kotlin--canary.207.3384012578.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.207.3384012578.kt)  
[typo_soulmate_react-native--canary.207.3384012578.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.207.3384012578.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.109.1-canary.207.3384012578.0
  npm install @salutejs/plasma-ui@1.143.1-canary.207.3384012578.0
  # or 
  yarn add @salutejs/plasma-temple@1.109.1-canary.207.3384012578.0
  yarn add @salutejs/plasma-ui@1.143.1-canary.207.3384012578.0
  ```
</details>

### CODE REVIEW

File: packages/plasma-ui/src/components/Pickers/Picker.tsx [hunk 1]
```diff
@@ -406,14 +407,18 @@ export const Picker = ({
     });
 
     useEffect(() => {
-        // Отключаем анимацию скролла при первом рендере
-        setScrollAnim(false);
+        // Отключаем анимацию скролла при первом рендере (React < 18)
+        if (!IS_REACT_18) {
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Pickers/Picker.tsx:411]: А почему так?

Comment (Yeti-or) [reply packages/plasma-ui/src/components/Pickers/Picker.tsx:411]: А я понял, это так было и больше не нужно

**Review by Yeti-or — COMMENTED** [review]
Выглядит норм, но перед влитием мне надо понять что с тестами всё окей

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-207/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-207/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-207/
  ui storybook: http://plasma.sberdevices.ru/pr-207/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-207/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-207/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-207/temple-storybook/

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-207/

**Review by LamaEats — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-207/
  ui storybook: http://plasma.sberdevices.ru/pr-207/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-207/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-207/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-207/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-207/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-207/
  ui storybook: http://plasma.sberdevices.ru/pr-207/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-207/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-207/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-207/temple-storybook/

**Review by Yakutoc — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**